paddle/fluid/operators/reader/blocking_queue.h

```diff
@@ -14,11 +14,14 @@
 
 #pragma once
 
+#include <gflags/gflags.h>
 #include <condition_variable>  // NOLINT
 #include <deque>
 
 #include "paddle/fluid/platform/enforce.h"
 
+DECLARE_bool(reader_queue_speed_test_mode);
```

Comment:
It seems that you can define `reader_queue_speed_test_mode ` here, because it is only used here.

Reply:
This is a header file, define the flag here will cause make error for duplicated symbols.